nova/compute/api.py

```diff
@@ -321,22 +321,27 @@ def _check_num_instances_quota(self, context, instance_type, min_count,
         vram_mb = 0
         req_ram = max_count * (instance_type['memory_mb'] + vram_mb)
 
+        quota_key_instances = 'instances'
+        if instance_type['extra_specs'].get('quota:separate', 'false') == 'true':
+            quota_key_instances = 'instances_' + instance_type['flavorid']
```

Comment:
Could we go here with `instance_type['name']` instead of `flavorid`? 
I think, `instances_x1.4xmemory` it is more descriptive than `instances_100`. There is a unique index on both the values.
Or am I missing something?

Reply:
I was not aware of the uniqueness constraint on the flavor name. I will use the name instead.